lib/web/auth/ldap/index.js

```diff
@@ -33,6 +33,13 @@ passport.use(new LDAPStrategy({
     username = user[config.ldap.usernameField]
   }
 
+  if (typeof uuid === 'undefined') {
```

Comment:
Maybe this should be moved up before the username code. If the username field is not used, this will say something like `Could not determine UUID for LDAP user "undefined"`.